### ISSUE TITLE
space not there

### ISSUE THREAD

**Issue by till-tomorrow** [coding-blocks/codingblocks.com#446]
![remocedot](https://user-images.githubusercontent.com/21954525/40187888-4953347a-5a16-11e8-85f1-5b10cde7789f.jpg)
 
It should be "Competitive Coding" not "Competitive.Coding"

In reference to #29

**Comment by shreyjain711** [comment]
@championswimmer does this repository have the resources folder? 
I couldn't find it. Please include it in case it isn't present. If it already exists then please point me towards it.

**Comment by Logan1x** [comment]
same @shreyjain711 can't find resources :(

**Comment by Logan1x** [comment]
Same @shreyjain711 i also could not find resources.

**Comment by shreyjain711** [comment]
@Logan1x  did u misspell it or you really found the typo in a markdown file? 😕 
Also, if possible, could you please tell me the file's name and it location?

**Comment by IamRaviTejaG** [comment]
@championswimmer I cannot seem to find the typo anywhere although it exists. Even the `grep` search doesn't return any results.

**Comment by Logan1x** [comment]
>you really found the typo in a markdown file?

yes, I really found it in md post. that means we need static site generator to generate and deploy the changes.
am I right @championswimmer ?

**Comment by shreyjain711** [comment]
@IamRaviTejaG is absolutely right. Although grep doesn't work with special characters but a query for "competitive" should give a list containing the error within it. But unfortunately, it didn't. 

@Logan1x  That is the real task yes, but I would say that if you think you have a lead and aren't able to solve the issue yourself, why not share it with others and see if anyone else can do it. This could even lead to you solving the issue before one of us. Or maybe the error isn't really where you think it is.

/src/resources should have been a separate folder. This [page](https://codingblocks.com/resources) has a different CSS than the rest of the website (since links aren't like that on other pages)

@championswimmer your help would help solve multiple issues. Please let us know where is the concerned page ([https://codingblocks.com/resources](https://codingblocks.com/resources)) located in the repository

**Comment by Logan1x** [comment]
@hereisnaman its been 3 days since @championswimmer added a label on this issue but not replying us. he might be busy so tagging you to forward us to right direction.

and @shreyjain711 you're looking at the wrong repo.coding block is org and it has a lot of repo where resources folder can be present.

>hould have been a separate folder

you were right in this line. try searching for resources repo in coding blocks.

**Comment by hereisnaman** [comment]
@Logan1x The `resource/*` is a separate project, a static site generator based on Jekyll. Here is the repo https://github.com/coding-blocks/resources

**Comment by till-tomorrow** [comment]
@championswimmer done! have a look.

**Comment by Sushant2** [comment]
Is this issue still open?
i would like to work on this!
please assign it to me!
